cliquet/resource.py

```diff
@@ -753,6 +755,15 @@ def _add_timestamp_header(self, response, timestamp=None):
         # Return timestamp as ETag.
         response.headers['ETag'] = encode_header('"%s"' % timestamp)
 
+    def _add_cache_header(self, response):
+        """Add Cache-Control and Expire headers, based a setting for this
+        resource.
+        """
+        key = 'cliquet.%s_cache_expires_seconds' % self.context.resource_name
```

Comment:
So we always have the same policy for all GETs on the same resource (collection get and resource get). We might want to add the `resource_type` to the configuration otherwise.